test/e2e/scoped_client_test.go

```diff
@@ -106,35 +106,36 @@ var _ = Describe("Scoped Client", func() {
 			return
 		}
 
-		if tt.grant != nil {
```

Comment:
Nit: In my experience, `tt` or `tc` (for test case) are common identifiers for rows in table-driven tests. I'm not sure this test is made more readable by changing it.

Reply:
kept it as `tc`